public/app/plugins/datasource/graphite/module.ts

```diff
@@ -5,12 +5,7 @@ import { MetricTankMetaInspector } from './components/MetricTankMetaInspector';
 import { ConfigEditor } from './configuration/ConfigEditor';
 import { GraphiteDatasource } from './datasource';
 
-class AnnotationsQueryCtrl {
-  static templateUrl = 'partials/annotations.editor.html';
```

Comment:
make sure to remove the corresponding html file

Reply:
Removed :) [1eab44e](https://github.com/grafana/grafana/pull/50453/commits/1eab44e6cca3a17a40ac74eff803fd3eb78ecb77)